src/qComponents/constants/locales/zh.ts

```diff
@@ -0,0 +1,43 @@
+export default {
+  QInput: {
+    charNumber: '字符数'
+  },
+  QSelect: {
+    loading: '加载中...',
+    more: '未显示所有结果，请优化您的搜索',
+    noMatch: '无匹配数据',
+    noData: '无数据',
+    selectAll: '全选'
+  },
+  QTextarea: {
+    charNumber: '字符数'
+  },
+  QDatePicker: {
+    placeholder: '选择日期',
+    startPlaceholder: '开始',
+    endPlaceholder: '结束',
+    prevYear: '前一年',
+    nextYear: '后一年',
+    prevMonth: '前一月',
+    nextMonth: '前一月',
```

Comment:
prevMonth / nextMonth are the same

Reply:
Fixed

Reply:
great! thank you, I'll merge it for now to add some changes and test it in local machine